tests/scripts/multi-node/build-rook.sh

```diff
@@ -24,7 +24,10 @@ function fail_if {
 
 function purge_rook_pods {
   cd "$rook_kube_templates_dir"
+  # Older rook versions use resource type "pool", newer versions
+  # use resource type "cephblockpools.ceph.rook.io".
   kubectl delete -n rook-ceph pool replicapool || true
+  kubectl delete -n rook-ceph cephblockpools.ceph.rook.io replicapool || true
   kubectl delete storageclass rook-ceph-block || true
   kubectl delete -f kube-registry.yaml || true
   kubectl delete -n rook-ceph cluster rook-ceph || true
```

Comment:
On this line we also have `cluster` that was renamed to `cephcluster`.

Reply:
Fixed